Tests/UnitTests/Fatras/Physics/EnergyLossTests.cpp

```diff
@@ -40,8 +41,10 @@ BOOST_DATA_TEST_CASE(BetheBloch, Dataset::parameters, pdg, phi, lambda, p,
 
 BOOST_DATA_TEST_CASE(BetheHeitler, Dataset::parameters, pdg, phi, lambda, p,
                      seed) {
+  (void)pdg;
   Generator gen(seed);
-  ActsFatras::Particle before = Dataset::makeParticle(pdg, phi, lambda, p);
+  ActsFatras::Particle before =
+      Dataset::makeParticle(Acts::PdgParticle::eElectron, phi, lambda, p);
```

Comment:
If you want to test this only for electrons, you can just create a new dataset on the fly, i.e. instead of `Dataset::parameters`, use something like

```
BOOST_DATA_TEST_CASE(BetheHeitler, momentumPhi * momentumLambda * momentumAbs ^ rngSeed, phi, lambda, p, seed)
```

Reply:
right, done